secrets/spring-cloud-vault/README.md

```diff
@@ -55,3 +56,61 @@ $ curl -i -X DELETE http://localhost:8080/api/orders
     Content-Length: 0
     Date: Fri, 13 Apr 2018 21:50:43 GMT
 ```
+
+## Refreshing Static Secrets
+Spring has an actuator we can use to facilitate the rotation of static credentials. Example below.
+
+1. Create the old secret.
+```
+$ curl -i  \
+    --header "X-Vault-Token: root" \
+    --request POST \
+    --data '{"secret":"hello-old"}' \
+    http://localhost:8200/v1/secret/spring-vault-demo
```

Comment:
Same thing here, I'd change `http://localhost:8200` to be `${VAULT_ADDR}` and add `export VAULT_ADDR=http://localhost:8200` above so they can just set it to whatever their VAULT_ADDR actually is and they can copy/past moving for all commands moving forward.